pulsar-client-cpp/python/pkg/osx/osx-10.13-python2.7/Vagrantfile

```diff
@@ -20,9 +20,10 @@
 #
```

Comment:
Can you remove the 10.11 if they're not used

Reply:
it's already removed.